packages/office-ui-fabric-react/src/components/ComboBox/__snapshots__/ComboBox.test.tsx.snap

```diff
@@ -228,7 +228,7 @@ exports[`ComboBox Renders ComboBox correctly 1`] = `
             background-color: #eaeaea;
             color: #212121;
           }
-      data-is-focusable={false}
+      data-is-focusable={true}
```

Comment:
This and other snapshots below seem odd since these corresponding tests weren't modified. Shouldn't existing behavior be preserved?